plugin.xml

```diff
@@ -120,11 +120,6 @@
                 android:taskAffinity="$PACKAGE_NAME"
                 android:launchMode="singleTask"
                 android:theme="@android:style/Theme.Translucent.NoTitleBar">
-                <intent-filter>
-                    <action android:name="android.intent.action.VIEW"/>
-                    <category android:name="android.intent.category.DEFAULT"/>
-                    <data android:scheme="$WECHATAPPID"/>
-                </intent-filter>
```

Comment:
I left the other `intent-filter` down below for the `WXPayEntryActivity` component. We aren't using that. I guess we could remove it, but it makes no practical difference and this might not work anyhow.